src/app/components/Account404/Account404.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import DocumentTitle from 'react-document-title';
+import { Link } from 'react-router';
+
+import appConfig from '../../data/appConfig';
+
+const Account404 = () => (
+  <DocumentTitle title={`404 | ${appConfig.displayTitle} | NYPL`}>
+    <main id="mainContent" className="not-found-404">
+      <div className="nypl-full-width-wrapper">
+        <div className="nypl-row">
+          <div className="nypl-column-three-quarters">
+            <h1>404 Not Found</h1>
```

Comment:
I don't know that there is a well-thought-out reason for making this a 404. Suggestions? It could just be a regular page, without any reference to `404`.